usaspending_api/api_docs/markdown/endpoints.md

```diff
@@ -91,6 +90,7 @@ The currently available endpoints are listed in the following table.
 |[/api/v2/disaster/overview/](/api/v2/disaster/overview/)|POST| Overview of Disaster/Emergency funding and spending|
 |[/api/v2/disaster/recipient/business_category/loans/](/api/v2/disaster/recipient/business_category/loans/)|POST| Records of loan Disaster/Emergency funding data by dimension|
 |[/api/v2/disaster/recipient/business_category/spending/](/api/v2/disaster/recipient/business_category/spending/)|POST| Records of spending Disaster/Emergency funding data by dimension|
+|[/api/v2/disaster/recipient/count/](/api/v2/disaster/recipient/count/)|POST| Dimensions Count of Disaster/Emergency funding data|
```

Comment:
Minor: here and elsewhere, since it's referring to count of _one_ dimension (recipient) in this case, don't pluralize: 
> Dimensions Count of ...

to
> Dimension count  of ...

Reply:
done